spot-client/src/common/app-state/config/selectors.js

```diff
@@ -141,8 +166,34 @@ export function getLoggingEndpoint(state) {
 }
 
 /**
- * A selector which gets the list of meeting domains that are known to support
- * meetings on Jitsi-Meet deployments.
+ * A selector which gets the full list of domains which can be shown in the
+ * meeting view. The list is calculated based on not only on the explicit
+ * whitelist but other enabled features.
+ *
+ * @param {Object} state - The Redux state.
+ * @returns {string[]}
+ */
+export const getAllAllowedMeetingDomains = createSelector(
+    getMeetingDomainsWhitelist,
+    _getConfiguredJwtDomains,
+    isZoomEnabled,
+    (whitelistedDomains, jwtDomains, allowZoomDomains) => {
```

Comment:
I'm wondering why jwt domains and whitelisted domains are separate config items? do we have any use case for that?

Reply:
I see `jwtDomains` are only used in the calendar and this list merged together with the whitelisted domains, so it seems to could be just one config? Am I missing something? Are JWT domains used for anything else?

Reply:
Whitelisted domains includes all domains we wanted to be displayed in the calendar. JWT domains represents which of those domains we want to pass the JWT into the jitsi frame. It could be that the JWT is valid for Env A but spot is configured to join both Env A and Env B. If the JWT is passed into Env B there is an error and the meeting is not joined.

Reply:
Got it

Reply:
we need them separate then